x/participation/module_simulation.go

```diff
@@ -14,9 +14,10 @@ import (
 )
 
 const (
-	opWeightMsgParticipate = "op_weight_msg_participate"
-	// TODO: Determine the simulation weight value
-	defaultWeightMsgParticipate int = 100
+	opWeightMsgParticipate            = "op_weight_msg_participate"
+	defaultWeightMsgParticipate   int = 100
```

Comment:
```suggestion
	defaultWeightMsgParticipate   int = 50
```

Just a small suggestion coming from observations made on other module's weight (e.g. campaign, launch).
Also FYI I've set `defaultWeightMsgWithdrawAllocations` to 50